src/components/container/AllergiesList/AllergiesList.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from "react"
+import { TermInformation } from "../../presentational/LabResultWithSparkline/LabResultWithSparkline"
+import MyDataHelps from "@careevolution/mydatahelps-js"
+import { useMyDataHelps } from "../../../helpers/useMyDataHelps"
+import { Action, LoadingIndicator, UnstyledButton } from "../../presentational"
+import { faQuestionCircle } from "@fortawesome/free-solid-svg-icons"
+import { FontAwesomeSvgIcon } from "react-fontawesome-svg-icon"
+
+export interface AllergiesListProps {
+    previewState?: "default"
+    onViewTermInfo(termInfo: TermInformation): void
+    innerRef?: React.Ref<HTMLDivElement>
+}
+
+interface Allergy {
+    Substance: string;
+    TermInformation?: TermInformation;
+    Reactions: string[];
+}
+
+export default function (props: AllergiesListProps) {
+    const [allergies, setAllergies] = useState<Allergy[] | null>(null);
+
+    function load() {
+        if (props.previewState == "default") {
```

Comment:
I would define all the preview states in story files, so that they aren't mingled with component function logic.

Reply:
Previews are not unique to stories but are also used in the view builder - otherwise we would absolutely just include them in story files.